pytorch_lightning/utilities/distributed.py

```diff
@@ -145,9 +145,9 @@ def sync_ddp(
     if group is None:
         group = torch.distributed.group.WORLD
 
-    if reduce_op is None:
+    if reduce_op is None or (isinstance(reduce_op, str) and reduce_op.lower() == "sum"):
         reduce_op = torch.distributed.ReduceOp.SUM
-    elif isinstance(reduce_op, str) and reduce_op in ("avg", "mean"):
+    elif isinstance(reduce_op, str) and reduce_op.lower() in ("avg", "mean"):
         reduce_op = torch.distributed.ReduceOp.SUM
```

Comment:
You didn't write this, but why is `ReduceOp.SUM` used for both cases? It should be outside of the `if-elif`

Reply:
Good catch ! It can be simplified. We just need to divide by world size.